docs/ops/memory/mem_setup.md

```diff
@@ -30,104 +30,127 @@ Flink allows both high level and fine-grained tuning of memory allocation within
 * toc
 {:toc}
 
-The further described memory configuration is applicable starting with the release version *1.10*. If you upgrade Flink
-from earlier versions, check the [migration guide](mem_migration.html) because many changes were introduced with the *1.10* release.
-
-<span class="label label-info">Note</span> This memory setup guide is relevant <strong>only for task executors</strong>!
-Check [job manager related configuration options](../config.html#jobmanager-heap-size) for the memory setup of job manager.
+The further described memory configuration is applicable starting with the release version *1.10* for task executors and
+*1.11* for job executors. If you upgrade Flink from earlier versions, check the [migration guide](mem_migration.html)
+because many changes were introduced with the *1.10* and *1.11* releases.
 
 ## Configure Total Memory
 
-The *total process memory* of Flink JVM processes consists of memory consumed by Flink application (*total Flink memory*)
-and by the JVM to run the process. The *total Flink memory* consumption includes usage of JVM heap,
-*managed memory* (managed by Flink) and other direct (or native) memory.
+The *total process memory* of Flink JVM processes consists of memory consumed by the Flink application (*total Flink memory*)
+and by the JVM to run the process. The *total Flink memory* consumption includes usage of *JVM Heap* and *Off-heap*
+(*Direct* or *Native*) memory.
 
 <center>
-  <img src="{{ site.baseurl }}/fig/simple_mem_model.svg" width="300px" alt="Simple memory model" usemap="#simple-mem-model">
+  <img src="{{ site.baseurl }}/fig/process_mem_model.svg" width="300px" alt="Flink's process memory model" usemap="#process-mem-model">
 </center>
 <br />
 
-If you run Flink locally (e.g. from your IDE) without creating a cluster, then only a subset of the memory configuration
-options are relevant, see also [local execution](mem_detail.html#local-execution) for more details.
+The simplest way to setup memory in Flink is to configure either of the two following options:
+
+| &nbsp;&nbsp;**Component**&nbsp;&nbsp; | &nbsp;&nbsp;**Option for task executor**&nbsp;&nbsp;                                 | &nbsp;&nbsp;**Option for job executor**&nbsp;&nbsp;                                |
```

Comment:
I think it should be Flink Master according to https://ci.apache.org/projects/flink/flink-docs-stable/concepts/glossary.html#flink-master

Reply:
Let's also rename `task executors` into `TaskManager` so that it is also in accordance with https://ci.apache.org/projects/flink/flink-docs-stable/concepts/glossary.html#flink-taskmanager